include/gazebo_yarp_plugins/ControlBoardDriver.h

```diff
@@ -351,6 +355,13 @@ class yarp::dev::GazeboYarpControlBoardDriver:
     std::vector<GazeboYarpControlBoardDriver::PID> m_velocityPIDs;
     std::vector<GazeboYarpControlBoardDriver::PID> m_impedancePosPDs;
 
+
+    // std::map<std::string, gazebo::common::PID> m_gazeboPositionPIDs; /* position PID controllers for each joint, needed to call gazebo's setForce() method */
+    std::vector<gazebo::common::PID> m_gazeboPositionPIDs;
+    std::vector<gazebo::common::PID> m_gazeboVelocityPIDs; /* velocity PID controllers for each joint, needed to call gazebo's setForce() method */
+    std::vector<gazebo::common::PID> m_gazeboImpedancePosPIDs; /* impedance position PID controllers for each joint, needed to call gazebo's setForce() method */
```

Comment:
@traversaro on the `void gazebo::physics::JointController::Update()` the method `gazebo::physics::Joint::SetForce(unsigned int _index, double _effort)` uses the output of `double gazebo::common::PID::Update(double _error, common::Time _dt)` as the `_effort` and I thought our solution should follow the same approach... You're saying that we should compute this `_effort` with the `GazeboYarpControlBoardDriver::PID` that already exists?